multisig/proposals/arbitrumone/vip-018/commands.ts

```diff
@@ -0,0 +1,100 @@
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { LzChainId } from "src/types";
+
+import { ACM, Assets, CONVERTER_NETWORK, converters } from "./addresses";
+
+const { arbitrumone } = NETWORK_ADDRESSES;
+
+type IncentiveAndAccessibility = [number, number];
+
+interface AcceptOwnership {
+  target: string;
+  signature: string;
+  params: [];
+}
+
+interface CallPermission {
+  target: string;
+  signature: string;
+  params: [string, string, string];
+}
+
+export const grant = (target: string, signature: string, caller: string): CallPermission => {
+  const config: CallPermission = {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [target, signature, caller],
+  };
+
+  return config;
+};
+
+const incentiveAndAccessibility: IncentiveAndAccessibility = [0, 1];
+
+function generateAcceptOwnershipCommands(ConvertersArray: string[]): AcceptOwnership[] {
+  const acceptOwnershipCommandsArray: AcceptOwnership[] = [];
+
+  for (const converter of ConvertersArray) {
+    const config: AcceptOwnership = {
+      target: converter,
+      signature: "acceptOwnership()",
+      params: [],
+    };
+
+    acceptOwnershipCommandsArray.push(config);
+  }
+
+  return acceptOwnershipCommandsArray;
+}
+
+const generateSetConverterNetworkCommands = () => {
+  return converters.map(converter => ({
+    target: converter,
+    signature: "setConverterNetwork(address)",
+    params: [CONVERTER_NETWORK],
+  }));
+};
+
+const generateAddConverterNetworkCommands = () => {
+  return converters.map(converter => ({
+    target: CONVERTER_NETWORK,
+    signature: "addTokenConverter(address)",
+    params: [converter],
+    dstChainId: LzChainId.arbitrumone,
+  }));
+};
+
+function generateCallPermissionCommands(ConvertersArray: string[]): CallPermission[] {
+  const callPermissionCommandsArray: CallPermission[] = [];
+
+  for (const converter of ConvertersArray) {
+    const config1 = grant(
+      converter,
+      "setConversionConfig(address,address,ConversionConfig)",
+      arbitrumone.NORMAL_TIMELOCK,
+    );
+    const config2 = grant(converter, "pauseConversion()", arbitrumone.NORMAL_TIMELOCK);
+    const config3 = grant(converter, "resumeConversion()", arbitrumone.NORMAL_TIMELOCK);
+    const config4 = grant(converter, "setMinAmountToConvert(uint256)", arbitrumone.NORMAL_TIMELOCK);
```

Comment:
I think now that we have omnichain governance in place we should configure all the permissions as specified in the docs (including the permissions to pause/unpause the conversions by fast-track/critical/guardian):

<img width="893" alt="image" src="https://github.com/user-attachments/assets/a570659a-e41a-4520-af4e-41e4df2f6248">

Reply:
In the VIP-245 and VIP-248 these permissions were set for the BNB Chain deployment. They could be used as a reference
* https://github.com/VenusProtocol/vips/blob/main/vips/vip-245/vip-245/vip-245.ts#L79
* https://github.com/VenusProtocol/vips/blob/main/vips/vip-248/vip-248/vip-248.ts#L129-L139